pom.xml

```diff
@@ -102,7 +102,7 @@
     <commons-lang-version>2.6</commons-lang-version>
     <commons-lang3-version>3.5</commons-lang3-version>
     <commons-collections.version>3.2.2</commons-collections.version>
-    <cdap.version>6.9.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.8.0-SNAPSHOT</cdap.version>
```

Comment:
why are we changing cdap.version?

Reply:
reverted the change, Also we don't need sl4j in e2e profile so removed that. (92b33cd4d2adbaf518814671b59547122f89e782)